src/programs/city/defense.js

```diff
@@ -8,6 +8,19 @@ class CityDefense extends kernel.process {
   constructor (...args) {
     super(...args)
     this.priority = PRIORITIES_DEFENSE
+
+    // Pre-load all effect values.
+    if (!global.TOWER_DAMAGE_EFFECT || !global.TOWER_REPAIR_EFFECT || !global.TOWER_HEAL_EFFECT) {
+      global.TOWER_DAMAGE_EFFECT = []
+      global.TOWER_REPAIR_EFFECT = []
+      global.TOWER_HEAL_EFFECT = []
+
+      for (let i = 0; i < 50; i++) {
+        global.TOWER_DAMAGE_EFFECT[i] = this.calculateWithFallOff(TOWER_POWER_ATTACK, i)
+        global.TOWER_REPAIR_EFFECT[i] = this.calculateWithFallOff(TOWER_POWER_REPAIR, i)
+        global.TOWER_HEAL_EFFECT[i] = this.calculateWithFallOff(TOWER_POWER_HEAL, i)
+      }
+    }
```

Comment:
I think these should be defined in `constants` so they can be reused throughout the code.

Reply:
They should also get placed in `package.json` so the linter won't toss an error.